site/content/internal/desktop-release-process.md

```diff
@@ -0,0 +1,117 @@
+
+6. When the release process is finished, go to the `mattermost/desktop` GitHub repository and select **Releases**.
+7. There should be a draft release of your new release candidate. Click the Pencil icon to edit.
+8. Make sure it's checked off as a pre-release, then select **Publish Release**.
+
+## Generate additional release candidates
+
+If there are any bugs reported by QA, once they are fixed we will need to generate an additional release candidate to verify that any issues have been fixed.
+
+To generate additional release candidates, you'll simply run the following commands to kick it off:
+```
+git checkout -b release-X.Y
+./scripts/release.sh rc
+git push --follow-tags upstream release-X.Y:release-X.Y
+```
+
+You can then follow steps **5-8** above to make sure the release is published.
+
+## Mac App Store
+
+Once the final release candidate has been approved by QA, you will need approval from the Mac App Store. To do so, we create a special release for them:
+```
+git checkout -b release-X.Y
+./scripts/release.sh mas
+git push --follow-tags upstream release-X.Y:release-X.Y
+```
+
+### Submit to App Review
+
+In this case, you won't get any notifications from Mattermost or GitHub, as we are submitting directly to Apple at this point. From here, we will need to submit the app for review:
+
+1. Go to our app on [App Store Connect](https://appstoreconnect.apple.com/apps/1614666244/appstore) and log in with your credentials.
+    - If you do not have access to App Store Connect, you'll have to ask your team lead.
+2. If there is not an unreleased version, select the **+** button in the top-left next to "macOS App", and fill out the version number (should just be `X.Y.Z`)
+3. Copy the changelog for this version into the **"What's New in This Version"** section.
+4. In the Build section, select **Add Build**. Find the build corresponding to the version that was just built. 
+    - You should be able to recognize it by timestamp, but if not, grab the build from TestFlight and verify that the version number is correct.
+5. Make sure under Version Release that "Manually release this version" is selected.
+6. Once that's all done, you can select the **Add for Review** button and follow the prompts to submit a review.
+
+If the app is **approved** by App Review, you will get an email from App Store Connect saying "Your submission was accepted". From there, you can go back and release the app once the final release has been cut.
+
+If the app is **rejected** by App Review, you will get an email from App Store Connect saying "We noticed an issue with your submission". At that point, you'll need to log back into App Store Connect and review their comments. Make the necessary changes and you can follow the same process as above to re-submit for review until the app has been approved.
```

Comment:
How long do they usually take for an answer?

Reply:
@devinbinnie - friendly bump on Maria's question.

Reply:
Sorry - missed this :D

Usually about 24 hours is their turnaround time, depends on the time of day. I'll add a note in here about that.